astropy/io/fits/convenience.py

```diff
@@ -98,7 +99,7 @@ def getheader(filename, *args, **kwargs):
 
     kwargs
         Any additional keyword arguments to be passed to
-        `astropy.io.fits.open`.
+        `~astropy.io.fits.open`.
```

Comment:
When I suggested that you add a `~`, I did not mean *all* API references but those that have unique names. :sweat_smile:  For this particular case, just showing `open()` can actually be confusing because many classes have this method but with different API. Can we revert this one? Sorry for the confusion.